www/public/different-names/index.html

```diff
@@ -8,7 +8,7 @@
   <title>Different Names in Roc</title>
   <meta name="description" content="Names of things in Roc that differ from other languages.">
   <meta name="viewport" content="width=device-width">
-  <link rel="stylesheet" href="/homepage.css">
+  <link rel="stylesheet" href="site.css">
```

Comment:
Nice catch!

FYI, in general it's safest to have stylesheet links start with `/` so they always load `site.css` from the domain root - otherwise they might 404 depending on the URL of the page they're on 😄 

```suggestion
  <link rel="stylesheet" href="/site.css">
```